packages/core/hooks/useFileAccessContextMenu.ts

```diff
@@ -147,37 +147,39 @@ export default (filters?: FileFilter[], onDismiss?: () => void) => {
                         dispatch(interaction.actions.downloadFiles());
                     },
                 },
-                {
-                    key: "move-files",
-                    text: "Move Files",
-                    title: "Move files between NAS Cache",
-                    disabled: !filters && fileSelection.count() === 0,
-                    iconProps: {
-                        iconName: "MoveToFolder",
-                    },
-                    subMenuProps: {
-                        items: [
-                            {
-                                key: "off-nas",
-                                text: "Off NAS Cache",
-                                title: "Move files off the NAS cache",
-                                onClick() {
-                                    // Placeholder for moving files off NAS Cache
-                                    console.log("Move files off NAS Cache");
-                                },
-                            },
-                            {
-                                key: "onto-nas",
-                                text: "Onto NAS Cache",
-                                title: "Move files onto the NAS cache",
-                                onClick() {
-                                    // Placeholder for moving files onto NAS Cache
-                                    console.log("Move files onto NAS Cache");
-                                },
-                            },
-                        ],
-                    },
-                },
+                ...(isQueryingAicsFms
+                    ? [
+                          {
+                              key: "move-files",
+                              text: "Move Files",
```

Comment:
Do we need the `Move Files` item and the sub-menu `Cache Location` now that there is only one `Cache Location` option?

There could just be a top-level `Move files to cache` item instead.

Reply:
resolved in [1bc859f](https://github.com/AllenInstitute/biofile-finder/pull/311/commits/1bc859fc0ecc021de5b18b02a10e656a50a92322)